src/index.css

```diff
@@ -42,6 +42,21 @@ button.command-block--pressed:hover {
     border: 2px dashed #000000;
 }
 
+button.btn.button--disabled svg {
+    stroke: #777;
+    fill: #777;
+}
+
+button.btn.button--disabled {
+    background-color: #FFF;
+    cursor: default;
+}
+
+button.btn.button--disabled:active:focus {
```

Comment:
The #FFF background-color is specific to `btn-primary`. Change the selector to: `button.btn-primary.button--disabled:active:focus`.